### PR TITLE
Temporary modification, heartbeat not counted for metric

### PR DESCRIPTION
**Pull request by taoyuanyuan**
### Issues associated with this PR

Your PR should present related issues you want to solve.

### Sign the CLA
Make sure you have signed the [CLA](https://www.clahub.com/agreements/alipay/sofa-mosn)

### Solutions
You should show your solutions about the issues in your PR, including the overall solutions, 
details and the changes. At this time, Chinese is allowed to describe these.

### UT result
Unit Test is needed if the code is changed, your unit test should cover boundary cases, corner cases, and some exceptional cases.
And you need to show the ut result.

### Benchmark
If your code involves the processing of every request, you should give the Benchmark Result

### Code Style
+ Make sure `Goimports` has run
+ Show `Golint` result

### CODE REVIEW

File: pkg/proxy/downstream.go [hunk 1]
```diff
@@ -226,6 +226,12 @@ func (s *downStream) cleanStream() {
 	s.proxy.listenerStats.DownstreamRequestTime.Update(streamDurationNs)
 	s.proxy.listenerStats.DownstreamRequestTimeTotal.Inc(streamDurationNs)
 
+	// todo: Temporary modification, heartbeat not counted
+	if s.requestInfo.IsHealthCheck() {
```
Comment (nejisama) [inline pkg/proxy/downstream.go]: 感觉不用先加再减，对于心跳直接不加了是不是可以就可以了？
